client/test-helpers/config/testing-library.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { render as rtlRender } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import initialReducer from 'calypso/state/reducer';
```

Comment:
Create a custom render function that wraps the passed component in a Redux provider. This allows us to test components that use `useSelector`, for instance.